pytri/__init__.py

```diff
@@ -518,8 +530,11 @@ def mesh(self, *args, **kwargs):
 
         geo.exec_three_obj_method("computeVertexNormals")
         color = kwargs.get("color", "#00bbee")
-
-        mesh = Mesh(geometry=geo, material=MeshLambertMaterial(color=color))
+        alpha = kwargs.get('alpha', 1.)
+        transparent = alpha != 1.
+        mat = MeshLambertMaterial(color=color, alpha=alpha, transparent=transparent)
+        mesh = Mesh(geometry=geo, material=mat)
         ret = self._add_layer(mesh)
-        self._reposition_camera_on_bbs()
+        if kwargs.get("auto_camera", False):
```

Comment:
Not to be a pain, but:

* If a user provides `auto_camera` to a function, it makes sense for the operation to be scoped to objects added in that function call
* If a user provides `auto_camera` to the _figure_ (this is what i was saying in the prev comment) it maybe makes more sense for the operation to be scoped to the full scene (as it is here as-is, I think?)